micro-ui/web/micro-ui-internals/packages/modules/core/src/components/TopBarSideBar/SideBar/SideBar.js

```
@@ -100,54 +133,105 @@ const Sidebar = ({ data }) => {
             // If the item has sub-items, render a dropdown with toggle button
-            const { leftIconArray, isDynamic } = extractLeftIcon(subItems) || {};
-            let leftIcon = IconsObject[leftIconArray] || IconsObject.collections;
-            if (isDynamic === "dynamic") {
-              var IconComp = require("@egovernments/digit-ui-react-components")?.[leftIconArray];
+            const { iconKey,iconName } = extractLeftIcon(subItems) || {};
+            let leftIcon =
+              IconsObject[iconName] || IconsObject.collections;
+            if (iconKey === "dynamic") {
+              var IconComp = require("@egovernments/digit-ui-react-components")?.[
+                iconName
+              ];
```

Biome (1.9.4)

[error] Use let or const instead of var.

A variable declared with var is accessible in the whole body of the function. Thus, the variable can be accessed before its initialization and outside the block where it is declared.
See MDN web docs for more details.
Unsafe fix: Use 'const' instead.


(lint/style/noVar)